Makefile

```diff
@@ -230,7 +222,7 @@ setup/moq:
 
 .PHONY: create/olm/bundle
 create/olm/bundle:
-	IMAGE_ORG=${IMAGE_ORG} IMAGE_REG=${IMAGE_REG} CHANNEL=${CHANNEL} UPGRADE=${UPGRADE} PREVIOUS_OPERATOR_VERSIONS=${PREVIOUS_OPERATOR_VERSIONS} ./scripts/bundle-rhmi-operators.sh
+	@CHANNEL=$(CHANNEL) PREV_VERSION=$(PREV_VERSION) PREVIOUS_OPERATOR_VERSIONS=$(PREVIOUS_OPERATOR_VERSIONS) ./scripts/bundle-rhmi-operators.sh
```

Comment:
@adam-cattermole small nit, could we update the name of the script to not include the acronym `rhmi`?

Reply:
Updated to create-olm-bundle.sh